clusterloader2/pkg/tuningset/qps_load.go

```diff
@@ -14,32 +14,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ticker
+package tuningset
 
 import (
 	"time"
 
+	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/perf-tests/clusterloader2/api"
 )
 
-type ticker struct {
-	C     <-chan struct{}
-	timer *time.Timer
-}
+type qpsLoad api.QpsLoad
 
-func newTicker(ts *api.TuningSet) (*ticker, error) {
-	ch := make(chan struct{})
-	timer, err := newTimer(ch, ts)
-	if err != nil {
-		return nil, err
-	}
-	return &ticker{
-		C:     ch,
-		timer: timer,
-	}, nil
+func newQpsLoad(ts *api.TuningSet) TuningSet {
```

Comment:
this should take api.QpsLoad as as a parameter.

Reply:
Done.